code/modules/orbits/spaceship.dm

```diff
@@ -148,7 +157,8 @@ GLOBAL_VAR_INIT(current_orbit,STANDARD_ORBIT)
 		var/choice = alert(usr, "This will end the round. Are you certain you wish to leave any groundside marines behind?", "Escape Velocity", "Cancel", "Yes", "Cancel")
 		if(choice != "Yes")
 			return
-		message_admins("[ADMIN_TPMONTY(usr)] Is going to finish the round via the spaceship orbits mechanic")
+		message_admins("[ADMIN_TPMONTY(usr)] Is going to finish the round via the spaceship orbits mechanic. set GLOB.current_orbit to 4 to prevent this.")
+		priority_announce("The coward hosts are attempting to flee!", "Prey is escaping!", ANNOUNCEMENT_REGULAR, 'sound/voice/alien_drool1.ogg', receivers = GLOB.alive_xeno_list + GLOB.observer_list)
```

Comment:
The tall hosts are escaping would be better

Reply:
OK